tests/e2e/Services/Teams/TeamsBaseServer.php

```diff
@@ -281,4 +282,101 @@ public function testDeleteUserUpdatesTeamMembershipCount($data)
         $this->assertIsInt($response['body']['total']);
         $this->assertEquals(true, $dateValidator->isValid($response['body']['$createdAt']));
     }
+
+    public function testTeamDeleteUpdatesUserMembership()
+    {
+        $new_users = [];
+
+        /**
+         * Create 5 new users and add their IDs to an array
+         */
+        for ($i = 0; $i < 5; $i++) {
+            $user = $this->client->call(Client::METHOD_POST, '/users', array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+            ], $this->getHeaders()), [
+                'userId' => ID::unique(),
+                'email' => 'newuser' . $i . '@localhost.test',
+                'password' => 'password',
+                'name' => 'New User ' . $i,
+            ], false);
+
+            $user_body = json_decode($user['body'], true);
```

Comment:
We typically don't call this 🧐 is it really needed?

Reply:
You mean the `json_decode` function? I saw a similar line in the create user test, so I decided to include it. If you're talking about the name, well, yeah it isn't really necessary. 

I also just saw that the `call` function takes a `decode` param, so I'm assuming that even if I want to decode the response, I can just do it by passing `true` instead of `false`?

Reply:
Ah yes, it looks like the decode param is optional and defaults to true so let's remove the false from the function call so it decodes automatically.

Reply:
I'm getting rid of that whole block and using the create membership endpoint for user creation anyways. So all good I guess!